llama_stack/providers/tests/vector_io/fixtures.py

```diff
@@ -53,6 +54,22 @@ def vector_io_faiss() -> ProviderFixture:
     )
 
 
+@pytest.fixture(scope="session")
+def vector_io_sqlite_vec() -> ProviderFixture:
```

Comment:
```suggestion
def vector_io_sqlite_vec(tmp_path) -> ProviderFixture:
```

How about using the `tmp_path` fixture instead?

Reply:
sounds good!

Reply:
If you’re writing tests within the pytest framework, tmp_path is generally more convenient due to its seamless integration, automatic cleanup, and the use of pathlib.Path objects.

It feels like a better pattern to me, but other maintainers should weight in :)

Reply:
Oh nice, yeah I can probably update them all in a follow up PR then, yeah? I think that would make sense but maybe it was intentionally ommitted previously.

Reply:
I'm cool with a followup PR :)